packages/xo-web/src/common/xo-line-chart/index.js

```diff
@@ -74,18 +74,17 @@ const makeLabelInterpolationFnc = (intl, nValues, endTimestamp, interval) => {
 }
 
 // Supported series: xvds, vifs, pifs.
-const buildSeries = ({ stats, label, addSumSeries }) => {
+const buildSeries = ({ stats, addSumSeries }) => {
   let series = []
 
   for (const io in stats) {
     const ioData = stats[io]
-    for (const letter in ioData) {
-      const data = ioData[letter]
-
+    for (const deviceName in ioData) {
```

Comment:
Is it correct for all supported series?

Reply:
Tested right now on other series, and no.